contracts/final_loop_library.scrypt

```diff
@@ -0,0 +1,88 @@
+// Library
+import "Util.scrypt";
+import "final_loop_import.scrypt";
+
+library Loop {
+	///////////////////////////////////////////////////////////
+	// Interesting part below
+	///////////////////////////////////////////////////////////
+	static function scriptCode(SigHashPreimage preimage): bytes {
+		//Return everything util.scriptCode except the space we need in this library
+		return Util.readVarint(preimage[104:len(preimage)-Aux.SizeOfObject-1]);
+	}
+	static function buildOutput(SigHashPreimage preimage, bytes outputScript, int outputSatoshis): bytes {

```

Comment:
Add empty line before each new function.

Reply:
@xhliu would it make sense to automate some of these checks with a linter? Or is there already one in place as part of the build / test process?

Reply:
Good idea. Will add one.